docs/client-sdk/client-sdk-porting-guide.md

```diff
@@ -37,11 +37,11 @@ The Client SDK reference implementation source code is organized as follows (fol
 ├── network* - - - - Network subsystem
 ├── NOTICES  - - - - License specific information
 ├── storage* - - - - Storage Subsystem
-└── utils  - - - - - Reference files for setting up TPM, flashing etc.
+└── utils  - - - - - Reference files for setting up TPM, flashing and others.
 ```
 
 ### Build System
-Starting with release v1.9.0 of the Client SDK, the build system uses cmake. This section explains the most prominent configurations in build system. The reference application is tightly coupled with the build system, storage and crypto APIs. The intent of this section is to cover those specific bindings to make the solution easy to traverse.
+Starting with release v1.9.0 of the Client SDK, the build system uses cmake. This section explains the most prominent configurations in build system. The reference application is tightly coupled with the build system, storage, and crypto APIs. The intent of this section is to cover those specific bindings to make the solution easy to traverse.
```

Comment:
Starting with release v1.9.0 of the Client SDK, the build system uses cmake. Can be restructured to "The build system uses cmake. "

Reply:
+1. Missed this out earlier.